docs-requirements.txt

```diff
@@ -7,9 +7,8 @@ sphinx-autodoc-typehints
 ./opentelemetry-api
 ./opentelemetry-semantic-conventions
 ./opentelemetry-sdk
-./opentelemetry-instrumentation
 
-# Required by ext packages
+# Required by instrumentation and exporter packages
```

Comment:
Do we still need these dependencies after instrumentation is moved away?

Reply:
I think they are needed for some of the examples.